<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sample List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="sampleListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="toolBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Tools</string>
        </property>
        <widget class="QComboBox" name="yComboBox">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>60</y>
           <width>181</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="xComboBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>181</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>68</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>X-Axis:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>30</y>
           <width>68</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>Y-Axis:</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>60</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Fix Axes</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="plotBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Plots</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gate List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>600</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionNew_Session"/>
    <addaction name="actionOpen_Session"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionLoad_Data_Files"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Graphs</string>
    </property>
   </widget>
   <addaction name="menuSession"/>
   <addaction name="menuData"/>
   <addaction name="menuGraph"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Session">
   <property name="text">
    <string>New Session</string>
   </property>
  </action>
  <action name="actionOpen_Session">
   <property name="text">
    <string>Open Session...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionLoad_Data_Files">
   <property name="text">
    <string>Load Data Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
